scripts/apps/contacts/components/fields/ContactName.tsx

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+const notEmpty = (str) => typeof str === 'string' && str.length > 0;
+
 export const ContactName: React.StatelessComponent<any> = ({item}) => {
-    const displayContact = item.first_name ? `${item.first_name} ${item.last_name}` : item.organisation;
+    const displayContact = notEmpty(item.first_name) || notEmpty(item.last_name) ?
+        `${notEmpty(item.first_name) ? item.first_name : ''} ${notEmpty(item.last_name) ? item.last_name : ''}`
```

Comment:
imo something like `[item.first_name, item.last_name].filter(notEmpty).join(' '),` would be better here so there is no extra space added, and maybe it's more readable?